CMakeLists.txt

```diff
@@ -330,7 +330,7 @@ install(
     DESTINATION "cmake"
     FILE "MbedTLSTargets.cmake")
 
-if(CMAKE_VERSION VERSION_GREATER 3.14)
+if(CMAKE_VERSION VERSION_GREATER 3.15 OR CMAKE_VERSION VERSION_EQUAL 3.15)
```

Comment:
Use `CMAKE_VERSION_GREATER_EQUAL`?

Reply:
I had thought about it. However `VERSION_GREATER_EQUAL` need CMake [3.7](https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html?highlight=greater_equal#genex:VERSION_GREATER_EQUAL), and the `cmake_minimum_required` in the `CMakeLists.txt` is [3.5.1](https://github.com/ARMmbed/mbedtls/blob/development/CMakeLists.txt#L21).